types/pino/index.d.ts

```diff
@@ -601,12 +601,9 @@ declare namespace P {
     type WriteFn = (o: object) => void;
 
     interface LogDescriptor {
-        pid: number;
-        hostname: string;
         level: number;
-        time: string;
+        time: number;
```

Comment:
`time` could also be a string, or it could even be undefined. See [here](https://getpino.io/#/docs/api?id=timestamp-boolean-function)

Reply:
I see the part that references a string (emphasized):
> If a function is supplied, it must synchronously return **a JSON string** representation of the time, e.g.,  `"time":1493426328206`

However, the example in that same line uses a number for the value, which seems to contradict the part about a string.

You are correct that it could be `undefined`! I missed that. The property should be optional to express this.